### PR TITLE
Add new MusicBrainz specific analyzers

### PR DESCRIPTION
**Pull request by samj1912**
These are adapted from Lucene's Classic Tokenizer and Filters

### CODE REVIEW

File: mb-solr/pom.xml [hunk 1]
```diff
@@ -129,6 +129,24 @@
 
 	<build>
 		<plugins>
+			<plugin>
+			  <groupId>de.jflex</groupId>
```
Comment (mineo) [inline mb-solr/pom.xml]: Nitpicking: the indentation here mixes tabs and spaces.

Comment (samj1912) [reply mb-solr/pom.xml]: Will fix.

**Comment by samj1912** [comment]
@mineo Yes, this is what it is addressing.

File: mb-solr/src/main/java/org/musicbrainz/search/analysis/MusicbrainzTokenizer.java [hunk 2]
```diff
@@ -0,0 +1,176 @@
+/*
```
Comment (mineo) [inline mb-solr/src/main/java/org/musicbrainz/search/analysis/MusicbrainzTokenizer.java]: The licene for most other files in this repo is MIT, this is Apache 2.0 code. This needs to either be documented somewhere or changed with permission of the authors.

**Pull request merged**